src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,40 @@
+package tests;
+
+import driver.DriverManager;
+import helpers.ConfigurationManager;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Listeners;
+import util.TestListener;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+@Listeners({TestListener.class})
+public class BaseTest {
+    protected static WebDriver driver;
+    protected final int WAIT_TIMEOUT_SECONDS =  Integer.parseInt(ConfigurationManager.getProperty("webdriver.timeout.seconds"));
```

Comment:
not sure it will work because @BeforeSuite with ConfigurationManager.loadProperties envokes later